modules/campaignion_overlay/campaignion_overlay.js

```diff
@@ -1,49 +1,12 @@
-/**
- * Provides a fade in type animation.
- *
- * The object is moved up from 5% below its actual `top` property
- * and its opacity is toggled.
- */
-function campaignionOverlayShow() {
-    var top_ = parseFloat($(this).css('top').replace('px', ''));
-    var move = $(window).height()/100 * 5;
-    var new_top = (top_+move).toString().concat('px');
-
-    $(this).css('top', new_top);
-    $(this).animate({
-      top: top_.toString().concat('px'),
-      opacity: "toggle"
-    }, 750, "easeInCubic");
-}
-
-/**
- * Provides a fade out type animation.
- *
- * The object is moved up to 5% above its actual `top` property
- * and its opacity is toggled.
- */
-function campaignionOverlayClose() {
-    var top_ = parseFloat($(this).css('top').replace('px', ''));
-    var move = $(window).height()/100 * 5;
-    var new_top = (top_-move).toString().concat('px');
-
-    $(this).animate({
-      top: new_top,
-      opacity: "toggle"
-    }, 750, "easeOutQuad", function() {
-      $(this).hide();
-    });
-}
-
 (function($) {
   Drupal.behaviors.campaignionOverlay = {
     attach: function(context, settings) {
       if (settings.campaignion_overlay
           && settings.campaignion_overlay.overlay_enabled) {
         // Add jQuery functions for animations
         jQuery.fn.extend({
-          campaignionOverlayShow: campaignionOverlayShow,
-          campaignionOverlayClose: campaignionOverlayClose
+          campaignionOverlayShow: Drupal.behaviors.campaignionOverlay.show,
```

Comment:
You can also just move them into the anonymous function closure if you don’t want them in the drupal behavior object.